### PR TITLE
Feature/user interface

### PR DESCRIPTION
**Pull request by uryelah**
- mock the basic flow of the game 
- add all actions needed to play Tic Tac Toe in the flow
- changed Xs and Os to symbols

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -1,3 +1,65 @@
 # frozen_string_literal: true
 
-'Hello World!'
+# !/usr/bin/env ruby
```
Comment (maelfosso) [inline bin/main.rb]: Please, move it at the beginning of the file.
It have to be the first instruction.

File: bin/main.rb [hunk 2]
```diff
@@ -1,3 +1,65 @@
...
+puts "Input player 1 name:\n"
+player1 = gets.chomp.capitalize
+puts "Input player 2 name:\n"
+player2 = gets.chomp.capitalize
+
+puts "Tic-tac-toe is a very fun game!\n#{player1} gets the 'X' and #{player2} the 'O'\nThe first one to connect three X's O's sequence is the Winner!\n"
+
+puts "Here's your Tic-tac-toe game board!\n"
+
+puts "    1  2  3\n a [#{houses[:a1]}][#{houses[:a2]}][#{houses[:a3]}]\n b [#{houses[:b1]}][#{houses[:b2]}][#{houses[:b3]}]\n c [#{houses[:c1]}][#{houses[:c2]}][#{houses[:c3]}]\n"
```
Comment (maelfosso) [inline bin/main.rb]: Great message !!!
However, how do I know what I should enter to play ?
Could you add an indication for that ?

File: bin/main.rb [hunk 3]
```diff
@@ -1,3 +1,65 @@
...
+turns = 1
+
+while turns <= 9
+  if turns.odd?
+    puts "#{player1} it's your turn! Choose a house!"
+    player1_choice = gets.chomp.downcase
+    if player1_choice == 'quit'
+      puts 'Quitting game'
+      break
+    end
```
Comment (maelfosso) [inline bin/main.rb:36]: According to all the message you display before, it's impossible for the user to know that he should type `quit` to quit the game

File: bin/main.rb [hunk 4]
```diff
@@ -1,3 +1,65 @@
...
+      redo
+    end
+    houses[player2_choice] = 'X'
+  end
+  puts "    1  2  3\n a [#{houses[:a1]}][#{houses[:a2]}][#{houses[:a3]}]\n b [#{houses[:b1]}][#{houses[:b2]}][#{houses[:b3]}]\n c [#{houses[:c1]}][#{houses[:c2]}][#{houses[:c3]}]\n"
+  turns += 1
+  if turns == 9 
+    puts "End of game"
+  end
+end
```
Comment (maelfosso) [inline bin/main.rb:120]: Please, where do you mock the winner ?

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Not good to be approved but you are on the right direction :smile: 

Hello @uryelah @majovainilla ,

Really good introduction in that milestone :+1: For a first try is really good :clap: 
However there are some changes that should be done.

1. I can't run the game. Here is what I'm getting from `bin/main.rb`
```ruby
$ bin/main.rb 
bin/main.rb: line 5: houses: command not found
bin/main.rb: line 7: puts: command not found
bin/main.rb: line 9: puts: command not found
bin/main.rb: line 10: player1: command not found
bin/main.rb: line 11: puts: command not found
bin/main.rb: line 12: player2: command not found
bin/main.rb: line 14: puts: command not found
bin/main.rb: line 16: puts: command not found
bin/main.rb: line 18: puts: command not found
bin/main.rb: line 20: turns: command not found
bin/main.rb: line 36: syntax error near unexpected token `/[OX]/'
bin/main.rb: line 36: `    if houses[player1_choice].match(/[OX]/)'
```
Read the comment below to fix it.

2. The game it's the TicTacToe. Could you, please, use **Cell/Case** instead of `House` ?

3.  Please, use **`:X / :O`** instead of 'X' / 'O'

4. Correct the messages you prompt

5. Read the comments below to know more about the others errors.

Kindly make the changes and submit a new code review request.
Thanks.

**Mael FOSSO**
[Github](https://github.com) | [Twitter](https://twitter.com) | [LinkedIn](https://www.linkedin.com/in/mael-elvis-fosso-650b6346/)

File: bin/main.rb (hunk 1 above)
Comment (uryelah) [reply bin/main.rb]: \ # !/usr/bin/env ruby  - at the first line  generated stickler issues

Comment (maelfosso) [reply bin/main.rb]: Seriously ? Write me on Slack to see that please.

Comment (uryelah) [reply bin/main.rb]: nvm, it was another branch

**Review by Ispirett — CHANGES_REQUESTED** [review]
### Status : Changes Reqiured 
Realy great work @uryelah  @majovainilla with the changes so far :+1:  :1st_place_medal: 

There are just a few bugs :bug:  to address. Now this does have to work perfectly.

**Required**
- [x] The game declares the victor, These to games it did not declare the winner.
```

    1  2  3
 a [O][X][ ]
 b [O][O][ ]
 c [O][X][X]

```

```    1  2  3
 a [O][X][ ]
 b [ ][O][ ]
 c [O][X][X]
```
```
    1  2  3
 a [O][X][O]
 b [O][X][ ]
 c [ ][X][ ]

```

#### Please remove the javascript and JSON file from the bin folder.

**Review by stratospherique — APPROVED** [review]
### Great work implementing the suggested changes :clap: . Congrats, you have passed the second milestone :heavy_check_mark: you may merge & move on to the third :+1: :+1: 

Keep up the good work :muscle: 
**Ahmed Mahfoudh**

###### PS: Please remove the javascript and JSON file from the bin folder.

**Pull request merged**